bindata/manifests/daemon/daemonset.yaml

```diff
@@ -35,8 +37,8 @@ spec:
                   fieldPath: metadata.namespace
             - name: EVENT_LOGGING_FILE
               value: "/tmp/ingress_node_firewall_events.log"
-          image: '{{.Image}}'
-          name: daemon
+            - name: POLL_PERIOD_SECONDS
+              value: "{{ .PollPeriodSeconds }}"
```

Comment:
I don't think we need to render this u define the default here and user can edit the manifest to change ?

Reply:
Agree.